### PR TITLE
refactor: Cleanup `LoadingScreen` and `ProgressBar`

### LINKED ISSUE #7182: [Task]: Cleanup `LoadingScreen` and `ProgressBar`

### PR DESCRIPTION
**Pull request by evavirseda**
## Summary

Closes #7182 

## Changelog

```
- Remove unnecessary LoadingScreen component
- Cleanup ProgressBar
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by VmMad — CHANGES_REQUESTED** [review]
Looks good but, the bug in the comment and wouldn't it be better to move the styles from the `ProgressBar` component to the styles? The lines there are so long...

**Review by VmMad — APPROVED** [review]
Nice

File: packages/shared/components/index.js [hunk 1]
```diff
@@ -8,7 +8,6 @@ export { default as Error } from './Error.svelte'
 export { default as Idle } from './Idle.svelte'
 export { default as Illustration } from './Illustration.svelte'
 export { default as Link } from './Link.svelte'
-export { default as LoadingScreen } from './LoadingScreen.svelte'
```
Comment (begonaalvarezd) [inline packages/shared/components/index.js:11]: This component got removed from there but its still present in the shared components folder, was the intention to move it somewhere else?

Comment (evavirseda) [reply packages/shared/components/index.js:11]: during some conflict resolution, it was added again :pray:

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**